ocemr/migrations/0006_add_visit_types.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2025-09-14 10:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ocemr', '0005_customizedtextfield'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='visit',
+            name='type',
+            field=models.CharField(choices=[(b'OUT', b'Outpatient'), (b'IN', b'Inpatient'), (b'MAT', b'Maternity')], default=b'OUT', max_length=3),
```

Comment:
We may want to leave MAT off the list as that needs a larger build out, but TBD.